packages/smart-accounts/tests/coinbase-account.test.ts

```diff
@@ -4,14 +4,17 @@ import {
   bundlerActions,
```

Comment:
Could we move this file within `smart-accounts/src/` and delete the `test/` folder? Right now this file doesn't compile, but CI doesn't detect that issue because of its path.